frontend/routes/manage/questionnaire/EditApplicant.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+    <div>
+        <div class="space-between">
+            <h1>{{applicant}}</h1>
```

Comment:
nit: spaces between braces and their contents

```suggestion
            <h1>{{ applicant }}</h1>
```